__tests__/app.test.js

```diff
@@ -269,4 +269,130 @@ describe("POST /api/reviews/:review_id/comments", () => {
          })
    });
    
+});
+
+describe.only("PATCH /api/reviews/:review_id", () => {
+  test("Get status 200 response and an object containing the updated review with votes added on", () => {
+  const newVotes = {
+        inc_votes: 20
+      };
+      return request(app)
+        .patch("/api/reviews/1")
+        .send(newVotes)
+        .expect(200)
+        .then((response) => {
+          expect(response.body.review[0].review_id).toBe(1);
+          expect(response.body.review[0].title).toBe("Agricola");
+          expect(response.body.review[0].category).toBe("euro game");
+          expect(response.body.review[0].designer).toBe("Uwe Rosenberg");
+          expect(response.body.review[0].owner).toBe("mallionaire");
+	        expect(response.body.review[0].review_body).toBe("Farmyard fun!");
+          expect(typeof response.body.review[0].review_img_url).toBe("string");
+	        expect(typeof response.body.review[0].created_at).toBe("string");
+	        expect(response.body.review[0].votes).toBe(21);
+          });
+  });
```

Comment:
For curcumstances like this, you might want to look at [.toMatchObject( )](https://jestjs.io/docs/expect#tomatchobjectobject) and [expect.objectContaining( )](https://jestjs.io/docs/expect#expectobjectcontainingobject) for future development.